cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -416,7 +416,7 @@ private void copyJarFile(URL jarURL, File file) {
     try {
       JarURLConnection jarConn = (JarURLConnection) jarURL.openConnection();
       try {
-        Files.copy(Resources.newInputStreamSupplier(jarConn.getJarFileURL()), file);
+        com.google.common.io.Files.copy(Resources.newInputStreamSupplier(jarConn.getJarFileURL()), file);
```

Comment:
It is better just open the stream

```java
  try (InputStream is = jarConn.getJarFileURL().openStream()) {
    Files.copy(is, file.toPath());
  }
```

Reply:
The current code also closes the `jarConn.getJarFile().close()`.
Do we still need to do that?